app/population/view/daily.njk

```diff
@@ -0,0 +1,116 @@
+{% extends "layouts/base.njk" %}
+{% from "panel/macro.njk" import appPanel %}
+{% from "data/macro.njk" import appData %}
+
+{%  macro statsEntry(property, value) %}
+  {%  set key = "population::daily_view.labels." + property %}
+  {%  set label = t(key) %}
+
+  <h4 class="govuk-heading-s">{{ label }}</h4>
+  <p class="govuk-body">{{ value }}</p>
+{% endmacro %}
+
+{% macro statsGroup(headingProperty, stats) %}
+  {%  set key = "population::daily_view.labels." + headingProperty %}
+  {%  set label = t(key) %}
+
+  <h3 class="govuk-heading-m">{{ label }}</h3>
+
+  {%  for stat in stats %}
+    {{  statsEntry(stat.property, stat.value) }}
+  {% endfor %}
+{% endmacro %}
+
+{% macro transfersGroup(headingProperty, transfers) %}
+  {%  set key = "population::daily_view.labels." + headingProperty %}
+  {%  set label = t(key) %}
+
+  <h3 class="govuk-heading-m">{{ label }}</h3>
+
+  <p class="govuk-body">
+  {%  for transfer in transfers %}
+    {{ transfer.count }} from {{  transfer.establishment }}<br>
```

Comment:
After our discussion around counts I wonder if this macro can be removed in favour of `statsGroup` as it looks very similar apart from this line.

And if transfers now also display a number they will be the same. And a url could be pass in as a 3rd argument.

Reply:
Macro has been removed and replaced with usage of `statsEntry`